php/README.markdown

```diff
@@ -0,0 +1,35 @@
+# DNSimple API PHP Examples
+
+The scripts in this directory demonstrate how to use the DNSimple PHP API wrapper to connect to the DNSimple API.
+
+## Running
+
+First you will need to ensure PHP is installed and then run `php -v`.
+
+Once you install the dependencies, create a file called `token.php` and put the following into it:
+
+```php
+<?php
+
+$token = "your_token";
+```
+
+Where `your_token` is an account token you generated on the DNSimple site from an account screen. Most of the example
```

Comment:
We should mention that we need a sandbox token.

Reply:
Makes sense. I was following the way other examples are worded, but this makes a lot of sense!